package.json

```diff
@@ -8,7 +8,8 @@
     "start": "node index.js",
     "start-dev": "pushd minecraftservermanager && ./node_modules/.bin/react-scripts start && popd",
     "build": "pushd minecraftservermanager && ./node_modules/.bin/react-scripts build && popd",
-    "test": "pushd minecraftservermanager && ./node_modules/.bin/react-scripts test --env=jsdom && popd",
+    "testBackend": "./node_modules/mocha/bin/_mocha -u bdd --colors --reporter mochawesome --reporter-options reportDir=./src/tests/results src/test" ,
```

Comment:
Just "mocha -u bdd ..." is enough. This should also be changed to `"test" : ...`